internal/resource/configmap.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	serverConfigMapName = "server-conf"
+	ServerConfigMap     = "server-conf"
```

Comment:
I'm not sure I agree with this name change. The content of the variable is the server-configuration config map name, not the config map itself.

Reply:
Good point. I will change it!